plugins/generator-1.18.2/forge-1.18.2/templates/plant/plant.java.ftl

```diff
@@ -83,6 +82,14 @@ public class ${name}Block extends <#if data.plantType == "normal">Flower<#elseif
 		<#if !data.useLootTableForDrops && (data.dropAmount == 0)>
 		.noDrops()
 		</#if>
+		<#if data.isSolid>
+		.noOcclusion()

```

Comment:
noOcclusion sounds like the opposite of solid, sure this is ok?

Reply:
Ok, so then dynamicShape controls the entity collisions?

Reply:
It's required on blocks with collision and offset, or the game crashes otherwise

Reply:
Ok, but what part is 1.18.2 code then makes plants block entity movement when isSolid is true?